index.html

```diff
@@ -40,11 +40,85 @@
     </style>
   </head>
   <body>
+
+    <header>
+      <h2>The Papillion</h2>
+      <nav>
+        <ul>
+          <li><a href="About">About</a></li>
+          <li><a href="Moreinformation">More info</a></li>
```

Comment:
Check out our examples on how we add an `h3` inside our `nav` elements

To be clear - what you have isn't wrong - it just could be better with the h3